src/k2/persistence/plog_service/PlogServer.h

```diff
@@ -39,24 +39,26 @@ Copyright(c) 2020 Futurewei Cloud
 namespace k2
 {
 
-// each PlogPage is a plog. It uses payload to store the data, and contains the sealed and offest as metadata
-struct PlogPage {
-    PlogPage(){
-        sealed=false;
-        offset=0;
-        payload = Payload(([] { return Binary(2 * 1024 * 1024); }));
-    }
-
-    bool sealed;
-    uint32_t offset;
-    Payload payload; 
-};
+
 
 class PlogServer
 {
 private:
     // the maximum size of each plog
-    const uint32_t PLOG_MAX_SIZE = 2 * 1024 * 1024;
+    const static uint32_t PLOG_MAX_SIZE = 16 * 1024 * 1024;

```

Comment:
very minor point, but you can use constexpr here

Reply:
fixed